src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1112,5 +1112,17 @@ public override void VisitRangeCaseClause(IRangeCaseClauseOperation operation)
                 AssertEx.Equal(new[] { operation.MinimumValue, operation.MaximumValue }, operation.Children);
             }
         }
+
+        public override void VisitFlowCapture(IFlowCaptureOperation operation)
+        {
+            Assert.Equal(OperationKind.FlowCapture, operation.Kind);
```

Comment:
`Assert.True(operation.IsImplicit)`?

Reply:
> Assert.True(operation.IsImplicit)?

The value is hardcoded, but I guess having an assert doesn't hurt, especially if something going to change in the future. Will add in the next PR.

---
In reply to: [163293701](https://github.com/dotnet/roslyn/pull/24358#discussion_r163293701) [](ancestors = 163293701)